### PR TITLE
Convert import step no1 to React [MAILPOET-1808]

### PR DESCRIPTION
**Pull request by pavel-mailpoet**

### CODE REVIEW

File: assets/css/src/components/_importExport.scss [hunk 1]
```diff
@@ -137,3 +120,22 @@ span {
     }
   }
 }
+
+.method_selection_step {
```
Comment (costasovo) [inline assets/css/src/components/_importExport.scss]: Please add mailpoet prefix here.

File: assets/js/src/subscribers/importExport/import/step_method_selection/method_paste.jsx [hunk 2]
```diff
@@ -1,19 +1,61 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import MailPoet from 'mailpoet';
+import ReactStringReplace from 'react-string-replace';
+
+const kbLink = 'http://docs.mailpoet.com/article/126-importing-subscribers-with-csv-files'
```
Comment (costasovo) [inline assets/js/src/subscribers/importExport/import/step_method_selection/method_paste.jsx]: This link redirects to https://kb.mailpoet.com/article/126-importing-subscribers-with-csv-files
so maybe we could update it

File: assets/js/src/subscribers/importExport/import/step_method_selection/method_upload.jsx [hunk 3]
```diff
@@ -1,21 +1,69 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import MailPoet from 'mailpoet';
+import ReactStringReplace from 'react-string-replace';
+
+const kbLink = 'http://docs.mailpoet.com/article/126-importing-subscribers-with-csv-files'
```
Comment (costasovo) [inline assets/js/src/subscribers/importExport/import/step_method_selection/method_upload.jsx]: We can update to https://kb.mailpoet.com/article/126-importing-subscribers-with-csv-files

**Comment by pavel-mailpoet** [comment]
Only the paste input method now preserves its value. When I tried to do it for paste input method I got:

`Uncaught DOMException: Failed to set the 'value' property on 'HTMLInputElement': This input element accepts a filename, which may only be programmatically set to the empty string.`

**Pull request merged**